optimum/onnxruntime/modeling_ort.py

```diff
@@ -1063,7 +1063,17 @@ def forward(
         pixel_values: Optional[Union[torch.Tensor, np.ndarray]] = None,
         input_features: Optional[Union[torch.Tensor, np.ndarray]] = None,
         input_values: Optional[Union[torch.Tensor, np.ndarray]] = None,
+        **kwargs,
     ):
+        # Handle return_dict from kwargs with default value False
+        return_dict = kwargs.pop("return_dict", False)
+
+        # Raise error for any unexpected kwargs
+        if kwargs:
+            raise ValueError(
+                f"{self.__class__.__name__} received {', '.join(kwargs.keys())}, but do not accept those arguments."
+            )
```

Comment:
hmm what about warning the user and directing them to use ORTModelForCustomTask if their model takes arbitrary tensor arguments.
```suggestion
            logger.warning(
                f"{self.__class__.__name__} received {', '.join(kwargs.keys())}, but do not handle those arguments. "
                "Please use `ORTModelForCustomTasks` if your model takes arbitrary or custom tensor arguments."
            )
```